cli/cppcheckexecutor.cpp

```diff
@@ -17,30 +17,44 @@
  */
 
 #include "cppcheckexecutor.h"
+
+#include <sys/_types/_sigaltstack.h>
```

Comment:
this looks like a posix include. please note that cppcheck is portable and should be compilable on visual studio etc also. we did not have it before and I have not seen compiler warnings.. so what problem does this solve?